--- a/conf.d/Makefile.am	2013-10-11 05:10:17.000000000 +0200
+++ b/conf.d/Makefile.am	2013-10-11 19:28:20.906699608 +0200
@@ -25,22 +25,7 @@ BUILT_SOURCES = README
 DOC_SOURCES = README.in
 DOC_FILES = $(DOC_SOURCES:.in=)
 
-CONF_LINKS = \
-	10-scale-bitmap-fonts.conf \
-	20-unhint-small-vera.conf \
-	30-urw-aliases.conf \
-	30-metric-aliases.conf \
-	40-nonlatin.conf \
-	45-latin.conf \
-	49-sansserif.conf \
-	50-user.conf \
-	51-local.conf \
-	60-latin.conf \
-	65-fonts-persian.conf \
-	65-nonlatin.conf \
-	69-unifont.conf \
-	80-delicious.conf \
-	90-synthetic.conf
+CONF_LINKS = 
 
 EXTRA_DIST = $(template_DATA) $(DOC_SOURCES)
 CLEANFILES = $(DOC_FILES)
@@ -83,21 +68,4 @@ template_DATA =				\
 README: $(srcdir)/README.in
 	sed "s|\@TEMPLATEDIR\@|$(templatedir)|" $< > $@
 
-install-data-hook:
-	mkdir -p $(DESTDIR)$(configdir)
-	@(echo cd $(DESTDIR)$(configdir);			\
-	  cd $(DESTDIR)$(configdir);				\
-	  for i in $(CONF_LINKS); do				\
-	    echo $(RM) $$i";" ln -s $(templatedir)/$$i .;	\
-	    $(RM) $$i;						\
-	    ln -s $(templatedir)/$$i .;				\
-	  done)
-uninstall-local:
-	@(echo cd $(DESTDIR)$(configdir);			\
-	  cd $(DESTDIR)$(configdir);				\
-	  for i in $(CONF_LINKS); do				\
-	    echo $(RM) $$i;					\
-	    $(RM) $$i;						\
-	  done)
-
 -include $(top_srcdir)/git.mk
--- a/conf.d/Makefile.in	2013-10-11 06:28:47.000000000 +0200
+++ b/conf.d/Makefile.in	2013-10-11 19:25:38.316697839 +0200
@@ -162,7 +162,7 @@ am__uninstall_files_from_dir = { \
     || { echo " ( cd '$$dir' && rm -f" $$files ")"; \
          $(am__cd) "$$dir" && rm -f $$files; }; \
   }
-am__installdirs = "$(DESTDIR)$(configdir)" "$(DESTDIR)$(templatedir)"
+am__installdirs = "$(DESTDIR)$(templatedir)"
 DATA = $(config_DATA) $(template_DATA)
 am__tagged_files = $(HEADERS) $(SOURCES) $(TAGS_FILES) $(LISP)
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
@@ -326,22 +326,7 @@ top_srcdir = @top_srcdir@
 BUILT_SOURCES = README
 DOC_SOURCES = README.in
 DOC_FILES = $(DOC_SOURCES:.in=)
-CONF_LINKS = \
-	10-scale-bitmap-fonts.conf \
-	20-unhint-small-vera.conf \
-	30-urw-aliases.conf \
-	30-metric-aliases.conf \
-	40-nonlatin.conf \
-	45-latin.conf \
-	49-sansserif.conf \
-	50-user.conf \
-	51-local.conf \
-	60-latin.conf \
-	65-fonts-persian.conf \
-	65-nonlatin.conf \
-	69-unifont.conf \
-	80-delicious.conf \
-	90-synthetic.conf
+CONF_LINKS = 
 
 EXTRA_DIST = $(template_DATA) $(DOC_SOURCES)
 CLEANFILES = $(DOC_FILES)
@@ -503,7 +488,7 @@ check: $(BUILT_SOURCES)
 	$(MAKE) $(AM_MAKEFLAGS) check-am
 all-am: Makefile $(DATA)
 installdirs:
-	for dir in "$(DESTDIR)$(configdir)" "$(DESTDIR)$(templatedir)"; do \
+	for dir in "$(DESTDIR)$(templatedir)"; do \
 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
 	done
 install: $(BUILT_SOURCES)
